### PR TITLE
Update What is indentation and why does it matter%3F.md

### PR DESCRIPTION
**Pull request by ChipNowacek**
Like @rogererens  PR, please just take this one as a suggestion. I don't expect or require anything. That Logseq is a powerful graph editor is a tricky thing to communicate. Your feedback on this PR would help me be of greater use to you in the future.

### CODE REVIEW

**Comment by ChipNowacek** [comment]
Just for emphasis: getting your feedback on this PR is far more important than seeing my suggestions adopted. Every day I'm more excited about what you have created and just want to help wherever I can.

File: pages/What is indentation and why does it matter%3F.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 	- ![parent-child-example-letters.png](../assets/parent-child-example-letters_1641572097841_0.png)
-- Let's walk through the example step-by-step:
+- Let's walk through the structure:
 	- Block **A** is a parent block; **B** and **C** are _children_ of **A**; block **D** is a _grandchild_ of **A**.
 	- Block **B** is a _child_ of **A** and it's in the same _branch_ as blocks **C** and **D**.
 	- Block **C** is a _child_ of **A** and the _parent_ of **D**.
-- Why is this important? As we've discussed in the previous lessons, Logseq is a networked note-taking tool that works with links. We also discussed that Logseq has no hierarchy, which is actually nuanced. As you've probably deduced by now, hierarchy in Logseq happens at the block level.
+- Why is this important? As we've discussed in the previous lessons, Logseq is a networked note-taking tool that works with links. Again, it's a [graph](https://en.wikipedia.org/wiki/Graph_(discrete_mathematics)). Graphs can contain hierarchies. A hierarchy is one type of a graph. Indentation allows us to quickly and visually create a small hierarchical section of your graph on a page.
```
Comment (logseq-cldwalker) [inline pages/What is indentation and why does it matter%3F.md:10]: This is a beginner friendly page and most of our users aren't technical so wikipedia links like this are more confusing than they are helpful

File: pages/What is indentation and why does it matter%3F.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
-title:: What is indentation and why does it matter?
+title:: Using Indentation
```
Comment (logseq-cldwalker) [inline pages/What is indentation and why does it matter%3F.md:1]: The title property can change the file name. This shouldn't be changed on collaborative graphs like this one

File: pages/What is indentation and why does it matter%3F.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
-- Indentation is one of the fundamental concept in Logseq (and outliners in general). It's the feature that allows block references and queries to work. In other words: indentation helps you to find stuff in Logseq.
-- The idea of indentation is simple: anytime you hit the `Tab` key, the block your cursor is in moves to the right. Indenting a block _nests_ it under the block above, creating a parent-child relationship:
+- Indentation is an important visual representation in Logseq (and outliners in general). It allows an view of a hierarchical section of your graph on a page. Representing hierarchies (mind maps, file directories, taxonomies, course prerequisites, org charts, etc) is an important visual tool. Logseq's indentation allows for this type of on-page presentation.
```
Comment (logseq-cldwalker) [inline pages/What is indentation and why does it matter%3F.md:3]: The original intro is focused on basic usage as this is meant to be beginner friendly. Replacing it with mentions of mind maps and hierarchies is going to confuse them and take the focus away from the tutorial's  intent

**Review by logseq-cldwalker — COMMENTED** [review]
@ChipNowacek Thanks for trying to contribute but unfortunately there isn't much we can use with this one. If you're looking for ways to help with documenting, documenting undocumented features would be amazing as described in https://github.com/logseq/docs/blob/master/CONTRIBUTING.md

**Pull request closed without merging**